src/core/Solana/jito/sendAndConfirmBundle.ts

```diff
@@ -0,0 +1,145 @@
+import type { SignatureResult, VersionedTransaction } from '@solana/web3.js'
+import bs58 from 'bs58'
+import { getJitoConnections } from '../connection.js'
+
+export type BundleResult = {
+  bundleId: string
+  txSignatures: string[]
+  signatureResults: (SignatureResult | null)[]
+}
+
+/**
+ * Send and confirm a bundle of transactions using Jito
+ * Automatically selects a Jito-enabled RPC connection and polls for confirmation
+ * across multiple Jito RPCs in parallel
+ * @param signedTransactions - Array of signed transactions
+ * @returns Bundle ID, transaction signatures, and confirmation results
+ */
+export async function sendAndConfirmBundle(
```

Comment:
This implementation diverges from how we handle standard transactions. Can we align it with the existing pattern so it's easier to maintain and follow?

1.	We should avoid running simulations inside a loop.
2.	The same loop should be used for connection handling, retrieving block height, confirming transactions, etc., rather than splitting the logic.